Parsers/ASimDns/ProductParsers/ASimDnsMicrosoftSysmon.yaml

```diff
@@ -15,88 +15,88 @@ References:
+            '<Data Name="ProcessGuid">{' ProcessGuid:string '}</Data>'
+            '<Data Name="ProcessId">' ProcessId:string '</Data>'
+            '<Data Name="QueryName">' DnsQuery:string '</Data>'
+            '<Data Name="QueryStatus">' DnsResponseCode:int  '</Data>'
+            '<Data Name="QueryResults">' DnsResponseName:string '</Data>'
+            '<Data Name="Image">' Process:string '</Data>'
+            '<Data Name="User">'SrcUsername:string '</Data>' *
+        | project-away EventData, ParameterXml, RenderedDescription
     };
-  let ParsedDnsEvent_WindowsEvent =() {
-    WindowsEvent
-    | where Provider == "Microsoft-Windows-Sysmon" and EventID == 22
-    | extend 
-       RuleName = tostring(EventData.RuleName),
-       EventEndTime = todatetime(EventData.UtcTime),
-       ProcessGuid = extract ('{(.*)}', 1, tostring(EventData.ProcessGuid), typeof(string)),
-       ProcessId = tostring(EventData.ProcessId),  
-       DnsQuery = tostring(EventData.QueryName),
-       DnsResponseCode = toint(EventData.QueryStatus),
-       DnsResponseName = tostring(EventData.QueryResults),
-       Process = tostring(EventData.Image)
-    | project-away EventData
-  };
-  union isfuzzy=true ParsedDnsEvent_Event, ParsedDnsEvent_WindowsEvent
-  | lookup RCodeTable on DnsResponseCode
-  | project-rename 
-      DvcHostname = Computer
-  | extend
-      EventOriginalUid = '22',
-      EventCount=int(1),
-      EventProduct = 'Sysmon',
-      EventVendor = 'Microsoft',
-      EventSchema = 'Dns',
-      EventSchemaVersion="0.1.2",
-      EventType = 'lookup',
-      EventResult = iff (DnsResponseCode == 0,'Success','Failure'),
-      EventStartTime = EventEndTime,
-      EventSubType= 'response',
-      EventSeverity= iif (DnsResponseCode == 0, 'Informational', 'Low'),
-      SrcUsernameType = 'Windows'
-  // -- Aliases
-  | extend 
-       EventResultDetails = DnsResponseCodeName,
-       Domain = DnsQuery,
-       Dvc = DvcHostname,
-       SrcHostname = DvcHostname,
-       DnsResponseCode = toint(iff (DnsResponseCode > 9000 and DnsResponseCode < 9100, DnsResponseCode-9000, DnsResponseCode))
-  // -- Backward Computability
-  | extend
-       Query=DnsQuery,
-       ResponseCodeName=DnsResponseCodeName 
+    let ParsedDnsEvent_WindowsEvent =() {
+        WindowsEvent
+        | where Provider == "Microsoft-Windows-Sysmon" and EventID == 22
+        | extend 
+            RuleName = tostring(EventData.RuleName),
+            EventEndTime = todatetime(EventData.UtcTime),
+            ProcessGuid = extract ('{(.*)}', 1, tostring(EventData.ProcessGuid), typeof(string)),
+            ProcessId = tostring(EventData.ProcessId),  
+            DnsQuery = tostring(EventData.QueryName),
+            DnsResponseCode = toint(EventData.QueryStatus),
+            DnsResponseName = tostring(EventData.QueryResults),
+            Process = tostring(EventData.Image),
+            SrcUsername = tostring(EventData.User)
+        | project-away EventData
```

Comment:
Just checking: No need to project-away ParameterXml, RenderedDescription in WindowsEvent?

Reply:
No, they do not appear in the schema